app/src/main/java/com/gmail/mxwild/newhabit/moviedetail/ActorViewModelFactory.kt

```diff
@@ -0,0 +1,13 @@
+package com.gmail.mxwild.newhabit.moviedetail
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+
+class ActorViewModelFactory : ViewModelProvider.Factory {
```

Comment:
You don't need factory for model with default constructor

Reply:
Delete